lector/resources/raw/io.github.BasioMeusPuga.Lector.metainfo.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 BasioMeusPuga <https://github.com/BasioMeusPuga> -->
+<!-- See <https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html> for an overview of the AppStream metainfo.xml spec. -->
+<component type="desktop-application">
+    <id>io.github.BasioMeusPuga.Lector</id>
+    <metadata_license>FSFAP</metadata_license>
```

Comment:
This element describes the license for the .metainfo.xml file itself, rather than anything to do with the rest of Lector. They specifically recommend permissive licenses, as it's just metadata and may be arbitrarily snipped and combined in various ways in the different app store category pages. I was unfamiliar with the [FSF All Permissive](https://spdx.org/licenses/FSFAP.html) license before reading the AppStream MetaInfo documentation, but it sounds a lot like an even shorter version of the MIT license, and it's what they recommend if you're undecided. You might or might not prefer some other license, though.

If you choose some other license, then the license ID should match match the identifiers used on the [SPDX License List](https://spdx.org/licenses/)